pkgs/tools/networking/cntlm/default.nix

```diff
@@ -21,11 +26,17 @@ stdenv.mkDerivation rec {
     description = "NTLM/NTLMv2 authenticating HTTP proxy";
     homepage = http://cntlm.sourceforge.net/;
     license = licenses.gpl2;
-    maintainers = 
+    maintainers =
       [
         maintainers.qknight
         maintainers.markWot
+        maintainers.carlosdagos
       ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ stdenv.lib.platforms.darwin;
   };
+} // stdenv.lib.optionalAttrs stdenv.isDarwin {
+  CXX = "clang++";
+  CC = "clang";
+  CPP = "clang -E";
+  CXXCPP = "clang++ -E";
```

Comment:
Should `CXX`/`CC` not that be set by default? If `CPP = "cpp"; CXXCPP = "cpp";` also works the platform condition could be removed.
cc @LnL7

Reply:
The CPP variants are not, the others are set by default yes.

Reply:
I think the best is do as @Mic92 points out and remove the platform condition. Thanks for reviewing!